patcher.sh

```diff
@@ -66,6 +66,12 @@ patch_am2r ()
 			exit 1
 		fi
 
+		# Check whether patchelf is installed
+		if [ !  -x "$(command -v patchelf)" ] ; then
```

Comment:
At this point, might as well create a function for it, rather than copy-pasting the same thing 4 times or so.